src/Core/Utilities/InternalDebounce/DebounceAction.cs

```diff
@@ -0,0 +1,61 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+namespace Microsoft.FluentUI.AspNetCore.Components.Utilities.InternalDebounce;
+
+/// <summary>
+/// The DebounceTask dispatcher delays the invocation of an action until a predetermined interval has elapsed since the last call.
+/// This ensures that the action is only invoked once after the calls have stopped for the specified duration.
+/// </summary>
+[Obsolete("Use Debounce, which inherits from DebounceTask.")]
```

Comment:
Why are we adding an obsolete class?

Reply:
It is an `internal` class. To use only the other class and avoid mistakes. But I would like to keep this class to validate the usage in future days. Next we will choose or delete this class